components/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/WorkflowImplService.java

```diff
@@ -54,6 +54,16 @@ void addBPSProfile(BPSProfile bpsProfileDTO, int tenantId)
      */
     void removeBPSProfile(String profileName) throws WorkflowImplException;
 
+    /**
+     * Remove all BPS profiles of given tenant id.
+     *
+     * @param tenantId Id of the tenant
```

Comment:
Change this as follows and check this throughout this PR
 @param tenantId of the tenant

Reply:
I have placed it there as, `tenantId` is the parameter name and `Id of the tenant` is the description. Hope that's fine.